src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/HomeTestResultCollectionViewCell.swift

```diff
@@ -85,4 +87,14 @@ class HomeTestResultCollectionViewCell: HomeCardCollectionViewCell {
 	@IBAction func primaryActionTriggered() {
 		delegate?.testResultCollectionViewCellPrimaryActionTriggered(self)
 	}
+
+	func setupAccessibility() {
+		titleLabel.isAccessibilityElement = true
+		subtitleLabel.isAccessibilityElement = true
+		descriptionLabel.isAccessibilityElement = true
+		illustrationView.isAccessibilityElement = true
+		isAccessibilityElement = false
+
+		titleLabel.accessibilityTraits = .header
+	}
```

Comment:
Could you please align with this one: https://github.com/corona-warn-app/cwa-app-ios/blob/db44967876567ca966a0431d1dc4cbe9230da66e/src/xcode/ENA/ENA/Source/Views/Home%20Screen/Cells/HomeTestResultCollectionViewCell.swift#L51

Reply:
Merged that in.